### ISSUE TITLE
[modding]scaling player models

### ISSUE THREAD

**Issue by jgregorytweedie** [yvt/openspades#1008]
a possibility to be able to scale player models for detailed modding, but have systems in place like with the guns where there are limits to how much you scale the model to prevent exploitation

**Comment by poVoq** [comment]
On a similar note, is it possible to have different head (& body) models for the three weapon classes?

**Comment by jgregorytweedie** [comment]
Yeah it is, a community member made a modded client of openspades called zerospades that does just that

**Comment by poVoq** [comment]
Google is not very helpful. Got a link to the Zerospades repo?
